### DISCUSSION TITLE
AJV 및 GitHub Action을 사용하여 JSON 파일에서 일관된 메타 정보 보장하기 - 정현수 기술 블로그

### DISCUSSION THREAD

**Discussion by giscus** [junghyeonsu/junghyeonsu.dev.comment#48] [Announcements]
# AJV 및 GitHub Action을 사용하여 JSON 파일에서 일관된 메타 정보 보장하기 - 정현수 기술 블로그

일관된 메타 정보로 데이터 무결성을 유지하기 위한 JSON 데이터를 처리할 때 오류를 사전에 방지하는 방법을 알아봅니다.

https://junghyeonsu.com/posts/using-ajv-and-github-actions-to-ensure-consistent-meta-information-across-json-files/

**Comment by tooooo1** [comment]
오 현수님 이번 글 너무 잘읽었어요! 좋은 글 감사합니다 ㅎㅎ

**Comment by okinawaa** [comment]
글 전개가 너무 자연스러워서 편하게 읽었습니다. 
JSON을 검증하여 휴먼에러를 방지할 수 있는 이런 좋은 방법을 알게되어서 기쁩니다😃
